src/Core/Logger.php

```diff
@@ -355,6 +355,6 @@ public static function devLog($msg, $level = LogLevel::DEBUG)
 
         $stamp1 = microtime(true);
         self::$devLogger->log($level, $msg);
-        self::getApp()->saveTimestamp($stamp1, "file");
+		self::getApp()->getProfiler()->saveTimestamp($stamp1, "file", System::callstack());
```

Comment:
Wrong indentation.

Reply:
The wrong indent was before, wasn't it?

Reply:
Yes, and the line before as well. Can you please check the whole file?

Reply:
btw. done for the complete file